cdc/metadata.cc

```diff
@@ -65,6 +65,13 @@ cdc::metadata::container_t::const_iterator cdc::metadata::gen_used_at(api::times
     return std::prev(it);
 }
 
+void cdc::metadata::clean_gens(api::timestamp_type ts) {
+    auto it = gen_used_at(ts - generation_leeway.count());
```

Comment:
The header comment says
```cpp
    /* Garbage-collects generations older than the generation operating at `ts`. */
```

but this is subtracting `generation_leeway`.

Also you're subtracting `generation_leeway` at call sites, so you're subtracting it twice

Reply:
This comment became outdated in v2.